packages/template/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "private": true,
-  "name": "@littlespoon/template",
+  "name": "template",
```

Comment:
This was changed to support the script that copies the template

https://github.com/little-spoon-dev/design-system/blob/c099a12/scripts/create-package.ts#L55-L58